### PR TITLE
Feat/manager 6076

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6076
| License          | BSD 3-Clause

## Description

Aims of this it's to inform customer that he can:
1. monitor his logs
2. and redirect him to right LDP

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js [hunk 1]
```diff
@@ -3,7 +3,13 @@ import head from 'lodash/head';
...
 import some from 'lodash/some';
 
-import { URLS, WEATHERMAP_URL } from './dashboard.constants';
+import {
+  URLS,
+  WEATHERMAP_URL,
+  ADVANCE_COMMERCIAL_RANGE,
+  GUARANTEED_BANDWIDTH_1GB_PLAN_CODES,
+  BARE_METAL_ADVANCED_WITHOUT_GUARANTEED_BW_TAG,
+} from './dashboard.constants';
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js:12]: please rebase on base branch

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+
```
Comment (radireddy) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js:2]: Move all code related to "up-sell cross-sell" to a separate component under instances.  This way its easy to remove or move the code. Refer to https://github.com/ovh/manager/pull/4205 for example

**Review by radireddy — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js]: done

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js]: done

File: packages/manager/modules/pci/package.json [hunk 3]
```diff
@@ -48,6 +49,7 @@
     "@ovh-ux/ng-translate-async-loader": "^2.1.0",
     "@ovh-ux/ng-ui-router-breadcrumb": "^1.1.1",
     "@ovh-ux/ng-ui-router-layout": "^4.0.1",
+    "@ovh-ux/ufrontend": "^1.1.0",
```
Comment (radireddy) [inline packages/manager/modules/pci/package.json:52]: you need this dependency?

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/pci/package.json (hunk 3 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/package.json:52]: Yes i need it to build URL.

Comment (ganeshkumar1989) [reply packages/manager/modules/pci/package.json]: This has been removed

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.html [hunk 4]
```diff
@@ -32,89 +32,100 @@
...
+                            data-translate="pci_projects_project_instances_instance_action_delete_label"
+                        ></span>
+                    </oui-tile-button>
+                </oui-tile>
+            </div>
+
+            <!-- Advices -->
+            <div class="mt-4">
+                <ovh-advices
+                    service-type="instance"
```
Comment (varun257) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.html:124]: This could be added as a constant instead.

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/instances/instance/instance.html:124]: I think it's ok because its only referred in one place

**Review by ganeshkumar1989 — APPROVED** [review]

**Pull request merged**